docs/tr2/README.md

```diff
@@ -310,6 +310,7 @@ as Notepad.
 - added Linux builds
 - added macOS builds
 - added .jpeg/.png screenshots
+- added BSON savegame support
```

Comment:
I think the meaning can be vague to the average recipient, instead I'd focus on the benefits for the TRLE and Rando communities both here and in the changelog.